src/App.ts

```diff
@@ -430,11 +436,11 @@ export default class App {
       // and theoretically, doing a fully custom (non express) receiver that implements OAuth
       usingOauth = true;
     }
-
     if (token !== undefined) {
-      if (authorize !== undefined || usingOauth) {
+      // If a token is supplied, the app is installed in at least one workspace
+      if (usingOauth || authorize !== undefined) {
         throw new AppInitializationError(
-          `token as well as authorize or oauth installer options were provided. ${tokenUsage}`,
+          `You provided a token along with either oauth installer options or authorize. ${tokenUsage}`,
```

Comment:
I read this error a new times over. For a developer receiving this message as error, would it be better to frame it as an error rather than a statement? For example: "You cannot provide a token with either oauth install options or authorize."